Assets/VRTK/Editor/VRTK_ObjectSetup.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+    using UnityEditor;
+    using VRTK.GrabAttachMechanics;
+    using VRTK.SecondaryControllerGrabActions;
+
+    public class VRTK_ObjectSetup : EditorWindow
+    {
+        private enum PrimaryGrab
+        {
+            ChildOfController,
+            FixedJoint,
+            Climbable,
+            CustomJoint,
+            RotatorTrack,
+            SpringJoint,
+            TrackObject
+        }
+        private enum SecondaryGrab
+        {
+            SwapControllers,
+            ControlDirection,
+            AxisScale
+        }
+        private bool useGrab = true;
+        private bool holdGrab = false;
+        private bool useUse = false;
+        private bool useIfGrabbed = false;
+        private bool holdUse = false;
+        private PrimaryGrab primGrab;
+        private SecondaryGrab secGrab;
+        private bool disableIdle = true;
+        private bool addrb = true;
+
+        [MenuItem("Window/VRTK/Setup Interactable Object")]
```

Comment:
I quite like the idea of having this as a main menu option

"VRTK/Setup Interactable Object"

thoughts?